### PR TITLE
Update accordion.js, closes #444

### LINKED ISSUE #444: Harden accordion Javascript against accordions that have no buttons
### Describe the bug, issue or concern
Accordions that exists but have not buttons currently crash the JS on the line:
`buttons[0].setAttribute("aria-expanded", "true")`

We should check if at least one button is available before trying to set it to expanded.

Im also not sure about the check for the aria-expanded value by just using getAttribute, as that also returns True for 'false' strings when used like this: `if(elem.getAttribute('aria-expanded'))`  on `<elem aria-explanded="false"></elem> `which is not what the code expects to do on line 48 of accordion.js

### To Reproduce
1. Create two accordions on one page
2. Make sure one has no buttons
3. Notice both have not initialized as expected.
4. See error console

### Expected behavior
Init all accordions as properly as possible

### Governance
- [x] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [x] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### PR DESCRIPTION
**Pull request by underdarknl**
These changes should close #444  for pages that by accident mess up their accordion code

### CODE REVIEW

**Review by ricklambrechts — APPROVED** [review]

**Comment by reinschaap** [comment]
nice nice

File: manon/accordion.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ function initAccordion(accordion) {
     }
 
     // Set up the initial `aria-expanded` state.
-    if (button.getAttribute("aria-expanded")) {
+    if (button.getAttribute("aria-expanded") === "true") {
```
Comment (ppvg) [inline manon/accordion.js:48]: Does this break the use case of explicitly setting `aria-expanded="false"` on the first button in the markup to stop it from being expanded by default?

File: manon/accordion.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ function initAccordion(accordion) {
   }
 
   // Expand the first item by default
-  if (!hasAriaExpandedMarkup) {
+  if (!hasAriaExpandedMarkup && buttons.length) {
```
Comment (ppvg) [inline manon/accordion.js:65]: 👍

File: manon/accordion.js (hunk 1 above)
Comment (underdarknl) [reply manon/accordion.js:48]: It should not,  and is more explicit, only an element with aria-expanded="true" should now be opened on page load.

Comment (ppvg) [reply manon/accordion.js:48]: PR: https://github.com/minvws/nl-rdo-manon/pull/636

**Pull request merged**